core/src/player.rs

```diff
@@ -2125,7 +2141,10 @@ impl PlayerBuilder {
             height: self.viewport_height,
             scale_factor: self.viewport_scale_factor,
         });
-        if let Some(movie) = self.movie {
+        if let Some(mut movie) = self.movie {
+            if let Some(url) = self.spoofed_url.clone() {
```

Comment:
I don't understand why this is needed given the similar logic in `core/src/loader.rs`. Maybe it can be removed (along with `SwfMovie::set_url()`)?

Reply:
This is used when the movie gets passed in from the PlayerBuilder, while the Loader core is used if it gets loaded from a Url.